R/get_map_legends.R

```diff
@@ -62,29 +65,34 @@
 #'   \item \code{color}: Recommended color for plotting the outcrop shape
 #'     element based on the dominant lithology.
 #'   \item \code{area}: The area of the outcrop shape element in
-#'   km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}}.
+#'     km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}}.
 #'   \item \code{tiny_area}: The area of the outcrop shape element in
-#'   km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "tiny" Macrostrat map
-#'   scale.
+#'     km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "tiny" Macrostrat map
+#'     scale.
 #'   \item \code{small_area}: The area of the outcrop shape element in
-#'   km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "small" Macrostrat map
-#'   scale.
+#'     km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "small" Macrostrat map
+#'     scale.
 #'   \item \code{medium_area}: The area of the outcrop shape element in
-#'   km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "medium" Macrostrat map
-#'   scale.
+#'     km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "medium" Macrostrat map
+#'     scale.
 #'   \item \code{large_area}: The area of the outcrop shape element in
-#'   km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "large" Macrostrat map
-#'   scale.
+#'     km\ifelse{html}{\out{<sup>2</sup>}}{\eqn{^2}} at "large" Macrostrat map
+#'     scale.
 #'   }
-#' @author Christopher D. Dean
-#' @section Reviewer: N/A
+#'
+#' @section Developer(s):
+#'   Christopher D. Dean
+#' @section Reviewer(s):
+#'   William Gearty
+#'
 #' @details Potential Macrostrat map scales are "tiny" (global), "small"
 #'   (continental), "medium" (regional), or "large" (local).
+#'
 #' @examples
 #' \dontrun{
-#'  ex1 <- head(get_map_legends(source_id = 1))
```

Comment:
I'd recommend including individual headings for each example. For this one, I'd go with: # Get legend information by ID of the original source map